site/components/header/index.js

```diff
@@ -10,15 +10,24 @@ const Header = () => {
   return (
     <header className={styles.header} role="banner">
       <a href="/" title="Home" className={styles.logo}>
+        <span className={styles.screenReaderText}>Red Badger</span>
         <InlineSVG src={logoSVG} />
       </a>
 
       <nav className={styles.mediumScreenNavContainer} role="navigation">
-        <ul className={styles.mediumScreenNav}>
-          <li><a href="/what-we-do/">What we do</a></li>
-          <li><a href="/about-us/">About us</a></li>
-          <li><a href="http://red-badger.com/blog/">Blog</a></li>
-          <li><a href="/about-us/events/">Events</a></li>
+        <ul role="listbox" className={styles.mediumScreenNav}>
+          <li role="listitem">
```

Comment:
It seems that unless we're using aria attributes on the `li`, there's no need to specify `role` attribute here since we have an underlying semantics conveying the same info to the screen reader.

http://stackoverflow.com/questions/11131234/the-reason-to-use-role-list-and-role-listitem